src/styles/main.css

```diff
@@ -0,0 +1,3 @@
+body {
+  margin: 0;
```

Comment:
Is it necessary?

Reply:
<img width="1680" alt="Screenshot 2024-01-02 at 12 38 02" src="https://github.com/nathan-hyan/cash-count-mvp/assets/74146741/81429e0b-1f81-426d-9886-1868abf05727">

I have to add it or else it gives me a default margin on the body.